crates/sound/src/audio_backends/pw_backend.rs

```diff
@@ -22,17 +25,80 @@ pub struct PCMParams {
     pub rate: u8,
 }
 
-// SAFETY: Safe as the structure can be sent to another thread.
-unsafe impl Send for WrapMainLoop {}
+#[derive(Clone)]
+pub struct PwThreadLoop(NonNull<pw_thread_loop>);
```

Comment:
This is not supposed to be used only here internally, right?
If it is the case, we could avoid `pub`.

Reply:
This struct being Clone is very probably not correct. It should be wrapped in an Arc/Rc instead.

Reply:
@stefano-garzarella it's not used here alone, I think it will be used in other functions like prepare() etc.